implementors.md

```diff
@@ -7,6 +7,7 @@ Registries currently supporting OCI Artifacts
 - [CNCF Distribution](https://github.com/distribution/distribution/) 
 - [Project Harbor](https://goharbor.io/blog/harbor-2.0/)
 - [Quay.io](https://www.openshift.com/blog/quay-oci-artifact-support-for-helm-charts)
+- [zot](https://zotregistry.io)
 _(to add your project, please open a [pull-request](https://github.com/opencontainers/artifacts/pulls))_
```

Comment:
@rchincha, 
Minor nit, can you add a blank line after the zot entry so the "(to add your project)..." is on the next line?

If you view the file in preview mode, you'll see the markdown formatting issue.